src/OpenLoco/src/World/StationManager.cpp

```diff
@@ -112,21 +112,21 @@ namespace OpenLoco::StationManager
         {
             if (station.stationTileSize == 0)
             {
-                station.var_29++;
-                if (station.var_29 != 5 && CompanyManager::isPlayerCompany(station.owner))
+                station.noTilesTimeout++;
+                if (station.noTilesTimeout == 5 && CompanyManager::isPlayerCompany(station.owner))
```

Comment:
noticed this mistake whilst labelling the offset. Its only meant to set the company emotion (sub_437F29) once not all the time.